tracing-subscriber/src/fmt/writer.rs

```diff
@@ -54,6 +54,31 @@ where
     }
 }
 
+/// Prints text using the standard library's macro `print!`. This writer allows stdout logs to be
+/// correctly captured by commands such as `cargo test`.
```

Comment:
Nit, take it or leave it: it might be nice to add a link to [the Rust book's section on capturing test output](https://doc.rust-lang.org/book/ch11-02-running-tests.html#showing-function-output) for new users who are unfamiliar with this concept?